project.yaml

```diff
@@ -74,8 +75,43 @@ dataSources:
           filter:
             module: balances
             method: Transfer
-        - handler: handleAllocationBatchCall
+        - handler: handleSponsorshipCreatePotCall
           kind: substrate/CallHandler
           filter:
-            module: allocations
-            method: batch
+            module: sponsorship
+            method: createPot
+        - handler: handleSponsorshipRegisterUsersCall
```

Comment:
Usually the EventHandlers are a better choice for indexing compared with the CallHandlers. The reason is when the call is nested for example inside a Utility batch, the subquery used to struggle to trace it. Is it still the case?

Reply:
True @aliXsed, we had issues with the batch in the past since it handle tons of data (e.x allocations)

Will update this on testnet, for use events instead of calls

Reply:
Will prepare a new PR when the sponsorship events got updated on mainnet, meanwhile pot indexing is desactivted